<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/数据分析.png</normaloff>images/数据分析.png</iconset>
  </property>
  <widget class="QPushButton" name="Button_login">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>250</width>
     <height>35</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>430</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 26pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>黑羽学院数据处理工具</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/数据分析.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
